### ISSUE TITLE
mailcatcher image is (too?) big

### ISSUE THREAD

**Issue by jbouzekri** [schickling/dockerfiles#56]
I have just installed the mailcatcher image and I am surprised by its size. Is it normal ?

```
$ docker images
REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE
schickling/mailcatcher         latest              07f9b78a9b71        6 days ago          745.3 MB
```

**Comment by schickling** [comment]
Not at all. Unfortunately I don't have enough time to look into this properly but what you could try to do is creating a PR replacing `ruby:2.2` by [alpine-ruby](https://hub.docker.com/r/frolvlad/alpine-ruby/). That should bring down the image < 100mb.

**Comment by jbouzekri** [comment]
Thanks. I will try that and I will look into the Dockerfile and try to understand what is happening.

**Comment by Lctrs** [comment]
Closed by #84.
